includes/Products.php

```diff
@@ -530,8 +530,18 @@ public static function update_google_product_category_id( \WC_Product $product,
 	 */
 	public static function get_product_gender( \WC_Product $product ) {
 
-		// TODO: implement
-		return '';
+		if ( $product->is_type( 'variation' ) ) {
+			$parent_product = wc_get_product( $product->get_parent_id() );
+			$gender         = $parent_product->get_meta( self::GENDER_META_KEY );
```

Comment:
`$parent_product` might not be an instance of `\WC_product` so we need to avoid that fatal in case an orphaned variation is passed in